source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -109,6 +109,39 @@
         });
     }
 
+    var StateManagerModel = function (data) {
+        var self = this;
+        this.evaluation = '';
+        this.posibleActions = ko.observableArray();
+        if (data) {
+            this.fromJs(data);
+        };
+        this.close = function () {
+            $('.content-evaluation').append($('.state-content-modal'));
+            $('.state-content-modal').hide();
+        };
+        this.revertEvaluationState = function (data) {
+            self.close;
+            viewmodel.isLoading(true);
+            var ajax = $.ajax("/Evaluations/api/RevertEvaluationState/" + self.evaluation.period + "/" + self.evaluation.userName + "?operation=" + data.ActionValue, {
+                type: "POST",
+                success: function (data) {
+                    viewmodel.isLoading(false);
```

Comment:
Maybe you can keep the loading state, it will disappear when current page is unloaded.